resources/measures/nighttime_oa_controls/tests/new_measure_test.rb

```diff
@@ -0,0 +1,323 @@
+             air_loop_oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+             min_oa_sched = air_loop_oa_system.minimumOutdoorAirSchedule.get()
+			 puts ('in simulated loop')
+			 if min_oa_sched.to_ScheduleRuleset.is_initialized
+			    puts('sched simulated loop')
+			    min_oa_sched = min_oa_sched .to_ScheduleRuleset.get
+			    puts(min_oa_sched)
+				assert(true)
+			end 
+			if min_oa_sched.to_ScheduleConstant.is_initialized
+			    puts('sched simulated loop')
+			    min_oa_sched = min_oa_sched.to_ScheduleConstant.get
+			    puts(min_oa_sched)
+				assert(false)
+			end 
+       end 
+      #end
+	  
+	  end 
+	  
+	  #confirm that schedule changed for unitary systems 
+	  li_unitary_systems.sort.each do |air_loop_hvac|
+
+      # change night OA schedule to match hvac operation schedule for no night OA
+        # Schedule to control whether or not unit ventilates at night - clone hvac availability schedule
+        next unless air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.is_initialized
+        air_loop_vent_sch = air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.get
+        air_loop_vent_sch.setName("#{air_loop_hvac.name}_night_novent_schedule")
+        if  air_loop_hvac.airLoopHVACOutdoorAirSystem.is_initialized
+		     air_loop_oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+             min_oa_sched = air_loop_oa_system.minimumOutdoorAirSchedule.get()
+			 puts ('in simulated loop')
+			 if min_oa_sched.to_ScheduleRuleset.is_initialized
+			    puts('sched simulated loop')
+			    min_oa_sched = min_oa_sched .to_ScheduleRuleset.get
+			    puts(min_oa_sched)
+				assert(true)
+			end 
+			if min_oa_sched.to_ScheduleConstant.is_initialized
+			    puts('sched simulated loop')
+			    min_oa_sched = min_oa_sched.to_ScheduleConstant.get
+			    puts(min_oa_sched)
+				assert(false)
+			end 
+			end 
+ 
+	 end 
+		
+	
+	
+	end 
+	
+	def test_361_retail_psz
+   
+   # this makes sure measure registers an na for non applicable model
+    osm_name = '361_Retail_PSZ_Gas_5a_mod_AHU_schedules_vent_added_output_v3.osm'
+    epw_name = 'TX_Port_Arthur_Jeffers_722410_16.epw'
+
+    osm_path = model_input_path(osm_name)
+	osm_path_output = model_output_path(osm_name)
+    epw_path = epw_input_path(epw_name)
+
+    # Create an instance of the measure
+    measure = NighttimeOAControls.new
+
+    # Load the model; only used here for populating arguments
+    model = load_model(osm_path)
+    arguments = measure.arguments(model)
+    argument_map = OpenStudio::Measure::OSArgumentMap.new
+	
+	# Apply the measure to the model and optionally run the model
+    result = apply_measure_and_run(__method__, measure, argument_map, osm_path, epw_path, run_model: true)
+	#result = apply_measure_and_run(__method__, measure, osm_path, epw_path, run_model: true)
+    assert_equal('Success', result.value.valueName)
```

Comment:
this test is asserting success - you want to assert 'na' for nonapplicable models.